### ISSUE TITLE
Test Home Page Images For Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4630]
# Page to Test:

Home page - https://openlibrary.org/

# Steps

Steps for this task are outlined in #4521 

# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by Sabreen-Parveen** [comment]
@bpmcneilly for the home page, books in the carousel do not have an alt atribute. I think #4651 will fix this too.

**Comment by Romi-776** [comment]
@bpmcneilly, @Sabreen-Parveen is right. 
All the cover images of the books shown on the home page doesn't have any alt attribute.
Also, the dropdown arrow images which are with browse and more button have an alt tag but doesn't assigned any value not even an empty string. Same thing is done with the archive logo in the footer of the page.

**Comment by Sabreen-Parveen** [comment]
@Romi-776 thanks for investigating this, but the alt="" justifies the footer logo acoording to comment https://github.com/internetarchive/openlibrary/issues/4521#issuecomment-772730099
and so is true for the dropdown arrow

**Comment by patricianicolentan** [comment]
Hi, I've submitted a pull request for adding alt text to the Open Library logo in the footer (following the header logo format) and the 'Around the Library' charts. The book cover images already had alt text. No other image accessibility violations on the home page.
